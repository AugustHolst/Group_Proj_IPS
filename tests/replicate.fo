fun bool writeBool(bool b) = write(b)

fun int writeInt(int d) = write(d)

fun [int] main() =
  let fs = replicate(7, false) in
  let ints = replicate(5, ~10) in
  let posints = replicate(3, 2) in
  let a = map(writeBool, fs) in
  map(writeInt, ints)