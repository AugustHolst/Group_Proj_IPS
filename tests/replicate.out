falsefalsefalsefalsefalsefalsefalse-10 -10 -10 -10 -10